### ISSUE TITLE
Environment Variables before running FlutterInstall

### ISSUE THREAD

**Issue by ChristianNavolskyi** [hey24sheep/azure-flutter-tasks#3]
Regarding the caching #2, it would also be nice to have some task to create the path variable before starting installation/downloading. 

The problem I have is, I have to hard code the sub path `/Flutter/2.2.3-stable/macos`. The version number I can extract and pass also to `FlutterInstall` but I would rather like to use simply the latest stable version and have the cache work for that too. Same for the platform (macos).

Would it be possible to have a task to create the environment variables `FlutterToolPath`, `FlutterPubCachePath` and `DartToolPath`. With them it would be easier to setup fitting cache-keys

**Comment by hey24sheep** [comment]
Hi @ChristianNavolskyi, I have updated the extension with a new task `Env` which will setup all three env vars. In respect to e08880c, I will close this issue. If you find any issue/bug with the new task, feel free to create a new issue regarding that.
